app/lib/actions/katello/environment/publish_repositories.rb

```diff
@@ -15,10 +15,12 @@ def plan(env, options = {})
           concurrence do
             repositories.each do |repository|
               sequence do
-                repository.set_container_repository_name
-                repository.clear_smart_proxy_sync_histories
-                plan_action(::Actions::Katello::Repository::InstanceUpdate, repository)
-                plan_action(::Actions::Katello::Repository::CapsuleSync, repository)
+                unless repository.root.is_container_push && repository.library_instance?
```

Comment:
Thoughts on changing the name of this action/file to specify `Container` since it's only called for those?

Reply:
There's an `unless` there, so it's for all repo types except container push ones that are in Default Organization View.

Reply:
This flow is called only for container repos though right? The only place this is called is in that flow and also I see `repository.set_container_repository_name` as the first thing in this action..

Reply:
The only place I see this getting called from passes Docker as content type. I used simple text search so I might have missed any other calls. Otherwise I think we should streamline that flow and update the name of the action to be more readable.

Reply:
I see what you mean, yeah this is totally a Docker specific action now. I wonder if once upon a time the action was used elsewhere.
I'll change it to `PublishContainerRepositories`